spec/predicates/README.md

```diff
@@ -21,6 +21,7 @@ our [vetting process], and may be of general interest:
     operations.
 -   [SLSA Verification Summary]: SLSA verification decision about a software
     artifact.
+-   [CycloneDX]: CycloneDX BOM for software artifacts.
```

Comment:
You'll need to add a link to [CycloneDX] below or else this will just show up as `[CycloneDX]` instead of as a real link.

Reply:
Sorry about that, I'll add it now. Thanks!